api/candlepin-api-spec.yaml

```diff
@@ -7914,6 +9529,61 @@ components:
             items:
               type: string
 
+    SystemPurposeComplianceStatusDTO:
```

Comment:
Although you have added specification but still using existing `org.candlepin.dto.api.v1.SystemPurposeComplianceStatusDTO`. Need to delete this existing DTO.

Reply:
Done